src/app/components/OptimoPromos/Lists/index.jsx

```diff
@@ -0,0 +1,21 @@
+import { GEL_SPACING_DBL } from '#app/legacy/gel-foundations/src/spacings';
+import styled from '@emotion/styled';
+
+// Bases
+export const StoryPromoLiBase = styled.li`
+  height: 100%;
+  margin-bottom: ${GEL_SPACING_DBL};
+  &:last-child {
+    margin: 0;
+  }
+`;
+
+export const StoryPromoUlBase = styled.ul`
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+`;
+
+StoryPromoUlBase.defaultProps = {
+  role: 'list',
```

Comment:
I feel like setting up default props and prop types shouldn't be in the styles file

Reply:
it is a very convenient way of dealing with some accessibility specs. I found it counterintuitive to create a new file just for this. Let me know what you think.

Reply:
I don't have a problem with creating a new file. It's not intuitive to have non styles stuff inside the styles file